### ISSUE TITLE
Suggestion - add to research 

### ISSUE THREAD

**Issue by keysivi** [nicolas-lang/Factorio.SpaceShuttle#9]
Good day!
Your mod is interesting, but not very well integrated into the game, as it is available initially.

It is also logical to add it to the research, so that it would be required to first study it, after the stages of the Cargo Plane and other recipes required for its manufacture.

Thanks!

**Comment by nicolas-lang** [comment]
Hi, 
Duplicate of https://github.com/nicolas-lang/Factorio.SpaceShuttle/issues/1 - and this should already be fixed.
Did you test with 0.0.4 ?

**Comment by keysivi** [comment]
Yes, just checked - the latest up-to-date version.

You're right - he is indeed in research. But all the same - at launch it is initially available for construction.

Forced to state a bug

**Comment by nicolas-lang** [comment]
confirmed - still broken.
I'll look into it.

**Comment by nicolas-lang** [comment]
Should be fixed in 0.0.5
I did not add "migrations" it will only be fixed if the "mods config" changes or if a new game is started.

if the issue persists please tell me
you can fix it in your save if you run this code
```
local function reload_tech_unlock(technology_name)
	log("reload_tech_unlock")
	for _, force in pairs(game.forces) do
		local unlock_state = false
		if force.technologies[technology_name].researched then
			unlock_state = true
		end
		for _, effect in pairs(force.technologies[technology_name].effects) do
			if effect.type == "unlock-recipe" then
				force.recipes[effect.recipe].enabled = unlock_state
				log(effect.recipe .. " enabled: " .. tostring(unlock_state))
			end
		end
	end
end
-------------------------------------------------------------------------------------
reload_tech_unlock("se-space-shuttle")
```

**Comment by keysivi** [comment]
Everything seems to be good. Thanks a lot!
